app/views/registrations/school_profile/show.html.erb

```diff
@@ -1,13 +1,19 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: registrations_question_two_path) %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">What is your school Unique Reference Number(URN)?</h2>
     <%= form_with(model: @school_profile_form, url: registrations_school_profile_path) do |f| %>
       <%= f.govuk_error_summary %>
-      <label class="govuk-hint">
-        This is a 6 digit number. Not sure? <a target='_blank' href='https://www.get-information-schools.service.gov.uk/'>Search for your school</a> on our register.
-        <%= f.govuk_text_field(:urn, width: 20, label:  nil, link_errors: true) %>
-      </label>
+      <%= f.govuk_text_field(
+            :urn,
+            width: 20,
+            label: -> { tag.h1("What is your school Unique Reference Number (URN)?", class: "govuk-heading-l") },
+            hint: -> { tag.p do
+              concat("This is a 6 digit number. Not sure? ")
+              concat(govuk_link_to("Search for your school", "https://www.get-information-schools.service.gov.uk/", target: "_blank"))
```

Comment:
Do we want some no-referrer and no-opener here?

In general, I wonder if there is a better way of specifying custom label, this is getting a bit annoying with all the concats. Not that I have any specific ideas :/ Maybe `render` method will work here? And we can have a partial for the label?

Reply:
> Do we want some no-referrer and no-opener here?

🤷‍♂️ this is from the old code. Happy to change it if that's required

> In general, I wonder if there is a better way of specifying custom label, this is getting a bit annoying with all the concats. Not that I have any specific ideas :/ Maybe render method will work here? And we can have a partial for the label?

I don't think I know enough about rails to really have an opinion on this, but having a partial just for this seems like overkill to me

Reply:
I think the no-referrer and no-opener aren't really needed, unless we expect `https://www.get-information-schools.service.gov.uk/` to try some naughty things on us - but at the same time, I don't see why we would like to allow them to interact with our page via javascript. 

Ech, I dislike programatically cobbling together html - might be just me. 3 lines aren't the end of the world though.